src/services/message.service.ts

```diff
@@ -70,6 +81,19 @@ export class MessageService {
       try {
         // Runs the sent function, with the data pulled from the emoji key.
         lambda(args.args);
+
+        embedData.emojiData = embedData.emojiData.filter((e) => e.emoji != reaction.emoji.name);
```

Comment:
What is this doing? This is resetting the value of a arg to the function, no?

Reply:
This removed the emote that was used from the reaction list.  So that if they remove and then add the emote back it won't run it again.  Plus it is also used to track if the embed message should be cut off or not.